internal/cliwrapper/cliwrapper.go

```diff
@@ -83,20 +96,19 @@ func (p *cliWrapper) Deploy(image string, podmanArgs []string, containerArgs []s
 }
 
 func (p *cliWrapper) KillAndClean(containerName string) error {
-	cmdKill := p.getPodmanCmd("kill", containerName)
-	p.logger.Debugf("Killing with command %v", cmdKill.Args)
-	if err := cmdKill.Run(); err != nil {
-		p.logger.Warningf("failed to kill pod %s, probably the execution terminated earlier", containerName)
+	_, err := p.runPodmanCmd("killing container "+containerName, "kill", containerName)
+	if err != nil {
+		p.logger.Warningf("failed to kill pod %s (%s); it may have exited earlier", containerName, err.Error())
```

Comment:
The term "pod" is incorrect -- `podman kill` acts on a single container, and we're not using `podman pod kill`.